proposals/NNNN-oracle-nio.md

```diff
@@ -0,0 +1,109 @@
+# Solution name
+
+* Proposal: [SSWG-NNNN](NNNN-oracle-nio.md)
```

Comment:
It think it will be [SSWG-0028](https://github.com/swift-server/sswg/blob/main/proposals/0028-oracle-nio.md). 
@FranzBusch ?

Reply:
Yes seems correct